openpdf/src/test/java/com/lowagie/text/pdf/SimpleBookmarkTest.java

```diff
@@ -15,7 +15,7 @@ class SimpleBookmarkTest {
     void testGetBookmarkWithNoTitle() throws IOException {
         InputStream is = getClass().getResourceAsStream("/OutlineUriActionWithNoTitle.pdf");
         PdfReader reader = new PdfReader(is);
-        List<?> list = SimpleBookmark.getBookmarkList(reader);
+        List<?> list = SimpleBookmark.getBookmark(reader);
```

Comment:
Usage of not-deprecated methods inside OpenPDF should be ok.

Reply:
reverted it